lib/generators/rodauth/install_generator.rb

```diff
@@ -71,9 +74,13 @@ def create_mailer
         end
 
         def create_fixtures
-          test_unit_options = ::Rails.application.config.generators.options[:test_unit]
-          if test_unit_options[:fixture] && test_unit_options[:fixture_replacement].nil?
-            template "app/test/fixtures/accounts.yml"
+          test_unit_options = ::Rails.application.config.generators.options[:test_unit]          
+          if test_unit_options[:fixture] && test_unit_options[:fixture_replacement].nil?                                    
+            if ::Rails.application.config.generators.options[:rails][:test_framework] == :rspec            
+              copy_file "app/test/fixtures/accounts.yml", "spec/fixtures/accounts.yml"               
+            else              
+              template "app/test/fixtures/accounts.yml", "test/fixtures/accounts.yml" 
```

Comment:
Should the source fixture continue to live in `templates/app/test/`, or wouldn't it make more sense to move it to `templates/test/` as well?

Reply:
> Should the template fixture continue to live in app/test/, or wouldn't it make more sense to move it to test/ as well?

Depends how you view the situation:  when I think `rodauth-rails/test` I think about tests for the `rodauth-rails` library itself, rather than templates for it. Pls confirm how you want to proceed?

Reply:
Note that my original comment was only talking about files within the `lib/generators/rodauth/templates/` directory.

Reply:
I see. So the fixtures, according to my tests in a real life rails app go in this directory:
Rspec Rails app: `spec/fixtures`
Other testing frameworks: `test/fixtures`  

The do not seem to be installing in the `app/tests` directory when a rrails app is being used. proof:

**Rspec**
![image](https://user-images.githubusercontent.com/15097447/179938446-dd9a0cf7-f74a-4b4c-9006-863f1c41a34e.png)
![image](https://user-images.githubusercontent.com/15097447/179938974-b7377e44-eb54-42b4-9e3d-c5fa71763002.png)

**test unit:**
![image](https://user-images.githubusercontent.com/15097447/179939251-ba01f22c-052e-431c-b7bd-63b72396c42f.png)

Reply:
Perhaps I am not understanding: please correct me if so?

Reply:
I'm talking about the templates directory inside rodauth-rails, not where the fixtures get imported into the app (that part was fixed). I don't know how else to explain it. We should not have an `app/test` directory anywhere, neither in rodauth-rails nor in Rails app when running `rodauth:install`.

Reply:
Ok now I understand! Apologies for my confusion. let me try for your proposal.

Reply:
https://github.com/janko/rodauth-rails/pull/127/commits/04717782c23534ab2f1b0c2e1451d07591974298 

Pls let me know if anything unresolved. i can squash and submit the final PR.